### DISCUSSION TITLE
Detekt scope

### DISCUSSION THREAD

**Discussion by IvanSanchez-uam** [detekt/detekt#5597] [Q&A] [answered]
In an android project there are some kind of directories , for instance there are (main, test, debug). Do you happen to know if detekt only check the .kt files under main?

**Comment by schalkms** [comment, marked as answer]
The checked source directories depend on your configuration, for instance in the Gradle task.

```Groovy
detekt {
    // The directories where detekt looks for source files. 
    // Defaults to `files("src/main/java", "src/test/java", "src/main/kotlin", "src/test/kotlin")`.
    source = files(
        "src/main/kotlin",
        "gensrc/main/kotlin"
    )
    ...
}
```

Source:
https://detekt.dev/docs/gettingstarted/gradle#options-for-detekt-configuration-closure
